state/protocol/badger/snapshot.go

```diff
@@ -161,9 +161,8 @@ func (s *Snapshot) Identities(selector flow.IdentityFilter) (flow.IdentityList,
 		return nil, err
 	}
 
-	// get identities from the current epoch first
-	identities := setup.Participants.Copy()
-	lookup := identities.Lookup()
+	// sort the identities so the 'Exists' binary search works
+	identities := setup.Participants.Sort(order.ByNodeIDAsc)
```

Comment:
Does this list already come "semi-sorted" on subsequent calls, or does it just get saved un-sorted, and we have to do the same sort every time?